addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -15,6 +15,269 @@ $MAX-Z-INDEX : 2147483647 !default;
+    position: relative;
+    top: -50px;
+}
+
+div .o_wslides_home_main {
+    .o_wslides_home_header {
+    }
+    .o_wslides_home_footer {
+    }
+
+    .o_wslides_home_aside {
+        .fa-check {
+            color: #00A09D;
+            font-size: 1.2em;
+        }
+    }
+    .o_wslides_home_content_section_title, .o_wslides_home_aside_title {
+        h3 {
+            font-size: 1.4em;
+        }
+        h4 {
+            font-size: 1.3em;
+        }
+    }
+
+    .o_wslides_course_card {
+        .card-body {
+            padding: 0.75em 1em;
+        }
+        .card-footer {
+            padding-top: 0.75em 1em;
+        }
+    }
+    // responsive card-deck like
+    .row.o_wslides_row > div[class*='col-'] {
+        display: flex;
+        flex:1 0 auto;
+        .card {
+            width:100%;
+        }
+    }
+
+    // tools
+    // ****************************************
+    .text_small_caps {
+        font-variant: small-caps;
+    }
+
+    .image-muted {
+        opacity: 0.5;
+    }
+
+    .o_wslides_home_circled {
+        border-radius: 50%;
+        width: 1.2em;
+        height: 1.2em;
+        padding: 0.25em 0.5em 0.25em 0.5em;
+        border: 0.2em solid #666;
+        text-align: center;
+        box-sizing: initial;
+    }
+
+    .progress {
+        overflow: visible;
+        margin-bottom: 0em;
+        height: 4px;
+        .progress-bar {
+            background-color: #00A09D;
+            position: relative;
+            border-radius: 4px;
+            span {
+                background-color: #00A09D;
+                position: absolute;
+                bottom: -20px;
+                font-size: 10px;
+                line-height: 10px;
+                padding: 2px 3px 2px 4px;
+                right: -1.4em;
+                border-radius: 2px;
+            }
+            span:after {
+                bottom: 100%;
+                left: 50%;
+                border: solid transparent;
+                content: " ";
+                height: 0;
+                width: 0;
+                position: absolute;
+                pointer-events: none;
+                border-color: rgba(255, 255, 255, 0);
+                border-bottom-color: #00A09D;
+                border-width: 5px;
+                margin-left: -5px;
+            }
+        }
+    }
+}
+
+// New course page
+// **************************************************
+
+.o_wslides_container {
+    margin: auto;
+    width: 100%;
+    max-width: 1140px;
+    padding: 32px 32px 0px 32px;
+}
+
+.o_wslides_course_nav {
+    margin: auto;
+    background: linear-gradient(150deg, #875A7B 20%, #62495B 80%) !important;
+    opacity: 0.95;
+
+    nav {
+        .breadcrumb-item::before {
+            font-weight: bold;
+            color: #FFFFFF;
+            opacity: 0.8;
+        }
+        .breadcrumb-item {
+            a {
+                font-weight: bold;
+                color: #FFFFFF;
+                opacity: 0.8;
```

Comment:
osef right now but just so you know it looks nice if you add something like `transition: opacity linear .3s;`